src/main/java/frameworks_and_drivers/Report_First_Screen.java

```diff
@@ -0,0 +1,82 @@
+package frameworks_and_drivers;
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.*;
+import java.io.IOException;
+import Database.InputBoundary;
+import Database.UserDataAccess;
+import Database.UserReporterInteractor;
+import Interface.UserReporterController;
+import Database.GroupRepoInt;
+import Interface.UserReporterPresenter;
+import usecase.UserFactory;
+import usecase.*;
+
+public class Report_First_Screen extends JFrame implements ActionListener {
+    public Report_First_Screen() {
+
+        JLabel title = new JLabel("Report System");
+        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+        JButton report = new JButton("Report User");
+        JButton block = new JButton("Block User");
```

Comment:
Since this function is not implemented, you can change it to a "back" Jbutton and go back to the last page